src/components/ErrorBoundary.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class ErrorBoundary extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { hasError: false };
+    }
+
+    static getDerivedStateFromError(error) {
+        // Update state so the next render will show the fallback UI.
+        return { hasError: true };
+    }
+
+    componentDidCatch(error, info) {
+        // Log error to the console
+        console.log(error, info);
```

Comment:
Make sure, you log any debug info to console only in dev mode(take a glance at DefineWebpackPlugin for it)

Reply:
done